style.css

```diff
@@ -0,0 +1,101 @@
+:root {
+  --main-radius: 5px;
+  --main-padding: 5px;
+}
+
```

Comment:
add this in your css :
```js
*{
  margin:0;
  padding:0;
  box-sizing:border-box;
};
```

Reply:
i would like to know in detail what is this used for?

Reply:
added